### ISSUE TITLE
CUSA03173 Bloodborne: Game crashes at first character movement

### ISSUE THREAD

**Issue by Virgula0** [shadps4-emu/shadPS4#1603]
Used version v0.4.1 https://github.com/shadps4-emu/shadPS4/releases/tag/Pre-release-shadPS4-2024-11-26-3f1be5a
The character creation works fine and even the intro video it's ok but when I do a few steps with the character, the emulator crashes.

## System settings

- GPU: RTX 3060 Ti 8Gb VRAM; Driver Version: 566.1.4 (latest, at the time of the issue)
- CPU: Intel i5 12600K
- RAM: 32Gb ddr4 3200Mhz
- OS: Windows 11 23H2

## Patches `Bloodborne.xml|shadPS4`

- Disable Chromatic Aberration ✅ 
- Disable Motion Blur ✅ 
- Disable HTTP Request ✅ 
- 60 FPS (With Deltatime) ✅ 
- Force Enable Old Hunters DLC ✅ 
- Disable vsync ✅ 
- Unlock Game Region ✅ 
- Enemy Control ✅ 

[Full file](https://github.com/user-attachments/files/17925588/wewe.txt)

## shadps4 settings

- Enabled fullscreen✅ 
- Is PS4 Pro ✅ 
- Width: `1280`, Height: `720 `
- Vblank Divider: `4`

## shadps4 crash log

Let me know if you need other info

[shad_log.txt](https://github.com/user-attachments/files/17925593/shad_log.txt)

## Controller

Xbox Series X controller via usb

**Comment by Virgula0** [comment]
Can be related somehow to: https://github.com/shadps4-emu/shadPS4/issues/787

**Comment by Missake212** [comment]
You should try again without any patches enabled, also disable the fullscreen and is ps4 pro option, better to check the "vanilla" game without anything conflicting with it.

**Comment by Virgula0** [comment]
> You should try again without any patches enabled, also disable the fullscreen and is ps4 pro option, better to check the "vanilla" game without anything conflicting with it.

Hi, thanks for your reply.

It happened something strange.
I disabled all patches, including fullscreen and ps4pro options yesterday when making some attempts with the emulator, and the emulator kept crashing. Then, I re-enabled some patches/options and since then, the emulator has been running fine!

I finally re-enabled all patches appointed above and the emulator no longer crashed.

At this point, I cannot say what the problem could be because the bug seems difficult to reproduce.

From crashing every time to running quite smoothly without apportioning any changes... can't say what it is but in my opinion, it should be investigated better because it can happen again for the same reason. I hope the crash file log can help.

P.S: i did not tried another prerelease, I was using the same exact version during all the attempts

**Comment by GHU7924** [comment]
If the problem concerns patches, then you can close the issue and put an issue on the repository of the patch.

**Comment by Virgula0** [comment]
> If the problem concerns patches, then you can close the issue and put an issue on the repository of the patch.

If the problem was caused by patches, it would have run without any problem disabling it. Read my latest message which describes better the situation. Maybe the reason is that I enabled patches at first run and you must at least enter in game before applying any patches. If this can be the reason the issue can be closed but again I don't really know the real reason.
